packages/default/scss/menu/_theme.scss

```diff
@@ -1,13 +1,20 @@
+$menu-bg: none !default;
+$menu-text: $link-text !default;
+$menu-hover-text: $link-hover-text !default;
+$menu-expanded-text: $base-text !default;
+$menu-selected-text: $list-item-selected-text !default;
+$menu-focused-shadow: $list-item-focused-shadow !default;
+
+$menu-scrolling-bg: $bg-color !default;
+$menu-scrolling-border: $widget-border !default;
+$menu-scrolling-icon: $link-text !default;
+$menu-scrolling-icon-hover: $link-hover-text !default;
+
 @include exports("menu/theme") {
-    $nav-text: $link-text !default;
```

Comment:
The same variables can be re-used for tabstrip, so I think that the $nav- variable is more appropriate.

Reply:
If so, they are an abstraction that should be declared in the `_variables` file, in order to be shared between components. After that, they should be propagated to the specific `$menu-*` variables.

That said, I'm not sure if such an abstraction is necessary at that point. It may be, if the variable values are the same across themes.